inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/sink/SQLServerSinkServiceTest.java

```diff
@@ -98,4 +106,68 @@ public void testGetAndUpdate() {
         deleteSink(sinkId);
     }
 
+    /**
+     * Just using in local test.
+     */
+    @Disabled
+    public void testDbResource() {
+        String url = "jdbc:sqlserver://127.0.0.1:1434;databaseName=inlong;";
+        String username = "sa";
+        String password = "123456";
+        String tableName = "test01";
+        String schemaName = "dbo";
+
+        try (Connection connection = SQLServerJdbcUtils.getConnection(url, username, password);) {
+            SQLServerTableInfo tableInfo = bulidTableInfo(schemaName, tableName);
+            SQLServerJdbcUtils.createSchema(connection, schemaName);
+            SQLServerJdbcUtils.createTable(connection, tableInfo);
+            List<SQLServerColumnInfo> addColumns = buildAddColumns();
+            SQLServerJdbcUtils.addColumns(connection, schemaName, tableName, addColumns);
+            List<SQLServerColumnInfo> columns = SQLServerJdbcUtils.getColumns(connection, schemaName, tableName);
+            Assertions.assertEquals(columns.size(), tableInfo.getColumns().size() + addColumns.size());
+        } catch (Exception e) {
+            // print to local console
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Build add SQLServer column info.
+     *
+     * @return {@link List}
+     */
+    private final List<SQLServerColumnInfo> buildAddColumns() {
+        List<SQLServerColumnInfo> addCloums = new ArrayList<>();
+        SQLServerColumnInfo test1 = new SQLServerColumnInfo("test1", "varchar(40)", "test1");
+        addCloums.add(test1);
+        SQLServerColumnInfo test2 = new SQLServerColumnInfo("test2", "varchar(40)", "test2");
+        addCloums.add(test2);
+        return addCloums;
+    }
+
+    /**
+     * Build test SQLServer table info.
+     *
+     * @param schemaName SqlServer schema name
+     * @param tableName SqlServer table name
+     * @return {@link SQLServerTableInfo}
+     */
+    private final SQLServerTableInfo bulidTableInfo(String schemaName, String tableName) {
+        SQLServerTableInfo tableInfo = new SQLServerTableInfo();
+        tableInfo.setTableName(tableName);
+        tableInfo.setComment("test01 ");
+        tableInfo.setPrimaryKey("id");
+        tableInfo.setSchemaName(schemaName);
+        List<SQLServerColumnInfo> columnInfos = new ArrayList<>();
```

Comment:
Initializing columnInfos should be separated from the field logic of set tableInfo, with a blank line in the middle, which will be more readable